docs/src/main/asciidoc/extension-metadata.adoc

```diff
@@ -97,12 +97,15 @@ metadata:
 description: "A JAX-RS implementation utilizing build time processing and Vert.x.\
   \ This extension is not compatible with the quarkus-resteasy extension, or any of\
   \ the extensions that depend on it." <4>
+scm:
+  url: "https://github.com/quarkusio/quarkus" <5>
 ----
 
 <1> Quarkus version the extension was built with
 <2> https://quarkus.io/guides/capabilities[Capabilities] this extension provides
 <3> Direct dependencies on other extensions
 <4> Description that can be displayed to users. In this case, the description was copied from the `pom.xml` of the extension module but it could also be provided in the template file.
+<5> The source code repository of this extension. In GitHub Actions builds, it will be determined automatically. For other github repos, it can be controlled by setting a `GITHUB_REPOSITORY` environment variable.
```

Comment:
Shouldn't we support things outside of GitHub? I don't really see why we should make it GitHub specific. I improved the wording in this direction but not sure about the `GITHUB_REPOSITORY` env variable. It might not be worth changing it but maybe a sentence saying it's not GH specific would help.
```suggestion
<5> The source code repository of this extension. In GitHub Actions builds, it will be determined automatically. For other repositories, it can be controlled by setting a `GITHUB_REPOSITORY` environment variable.
```

Reply:
Wonder if in the non-GitHub Actions case, it would be easier to let it be defined in the YAML file directly.

Reply:
The current implementation actually doesn't work for other repositories in general, only for other GitHub ones. (We prepend `http://github.com/` to what's set in the variable.) 

Obviously, this isn't great, and I'll be changing it by the end of next week or early next week so we can make use of the scm coordinates in the pom.xml. 

I see I got the capitalisation wrong on GitHub in the second sentence, but since this is such temporary prose I'm not sure it's worth fixing.

Reply:
> I see I got the capitalisation wrong on GitHub in the second sentence, but since this is such temporary prose I'm not sure it's worth fixing.

Yeah, I had it fixed locally before I added the comment so I just pushed what I had.